go/controller/api/v1alpha1/helmrepository_types.go

```diff
@@ -134,6 +139,9 @@ type HelmRepositoryAuthAWS struct {
 	// +kubebuilder:validation:Optional
 	SecretAccessKeySecretRef *corev1.SecretReference `json:"secretAccessKeySecretRef,omitempty"`
 
+	// +kubebuilder:validation:Optional
+	SecretKeyRef *corev1.SecretKeySelector `json:"secretKeyRef,omitempty"`
```

Comment:
this will be goofy, but following the pattern, `secretAccessKeySecretKeyRef`

Reply:
done